tests/test_evaluator.py

```diff
@@ -16,15 +16,15 @@
 
 from unittest import TestCase
 
-from datasets import Dataset, load_dataset, load_metric
-from transformers import AutoTokenizer, BertForSequenceClassification, pipeline
+from datasets import load_dataset
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, pipeline
```

Comment:
I took the liberty of cleaning this up a bit to follow the conventions in `transformers` (use autoclasses over model-specific ones where possible)